jsonnet/dashboards/grafana-templates.libsonnet

```diff
@@ -113,11 +113,13 @@ local template = grafana.template;
         query=query,
       ),
 
-    namespaceTemplate(query)::
+    namespaceTemplate(query, includeAll=null, multi=null)::
```

Comment:
Since in `baseTemplate` default values for `includeAll` and `multi` is `true` it should be default also here. Otherwise calls of `namespaceTemplate` where these params are not set would change their behaviour.